sale_stock_available_to_promise_release/models/stock_picking.py

```diff
@@ -0,0 +1,12 @@
+# Copyright 2020 Camptocamp (https://www.camptocamp.com)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import fields, models
+
+
+class StockPicking(models.Model):
+    _inherit = "stock.picking"
+
+    sale_date_expected = fields.Date(
+        string="Delivery Date", related="group_id.date_expected", store="True"
```

Comment:
I fixed this here https://github.com/OCA/wms/pull/73/files#diff-fca2e9865c2d75a9793dbe699bffdf976606d996b69d09cacd25660f2926e998R51 and copy-pasted the wrong version :facepalm: